src/applications/widget-editor/src/components/editor/component.js

```diff
@@ -167,9 +178,17 @@ class Editor extends React.Component {
   }, 1000);
 
   resolveAreaIntersection(areaIntersection) {
-    const { setFilters, hasGeoInfo } = this.props;
+    const {
+      setFilters,
+      hasGeoInfo,
+    } = this.props;
+    const { areaIntersection: prevAreaIntersection } = this.props.editorState.filters;
+
     if (areaIntersection && hasGeoInfo) {
       setFilters({ areaIntersection });
+    } else if (!areaIntersection && prevAreaIntersection && prevAreaIntersection.length > 0) {
+      // If we get no intersection from parent but we have one present, reset intersection selection
+      setFilters({ areaIntersection: "" });
```

Comment:
I would rather set it to `null` as this is the [default state](https://github.com/Vizzuality/widget-editor/blob/374694f85e995d72c336ed8d23380c2488a596bc/src/packages/shared/src/modules/filters/initial-state.js#L3) of the filter.